submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/FileListResource.kt

```diff
@@ -18,16 +22,27 @@ class FileListResource(
     private val sourceGenerator: SourceGenerator,
     private val fileListValidator: FileListValidator,
     private val onBehalfUtils: OnBehalfUtils,
+    private val submissionQueryService: SubmissionPersistenceQueryService,
```

Comment:
calling persistence layer directly from web layer is not good.  IMO we should include all this in FileListValidator which is the service/domain class in charge of this operation.